source/HockeySDK.iOSBindings/ApiDefinition.cs

```diff
@@ -6,671 +6,694 @@
-    public delegate NSString BITLogMessageProvider ();
-
-    // typedef void (^BITLogHandler)(BITLogMessageProvider, BITLogLevel, const char *, const char *, uint);
-    public unsafe delegate void BITLogHandler (BITLogMessageProvider messageProvider, BITLogLevel logLevel, IntPtr file, IntPtr function, uint line);
-
-    [Static]
-    partial interface Constants
-    {
-        // extern NSString *const kBITCrashErrorDomain;
-        [Field ("kBITCrashErrorDomain", "__Internal")]
-        NSString CrashErrorDomain { get; }
-    
-        // extern NSString *const kBITUpdateErrorDomain;
-        [Field ("kBITUpdateErrorDomain", "__Internal")]
-        NSString UpdateErrorDomain { get; }
-    
-        // extern NSString *const kBITFeedbackErrorDomain;
-        [Field ("kBITFeedbackErrorDomain", "__Internal")]
-        NSString FeedbackErrorDomain { get; }
-    
-        // extern NSString *const kBITAuthenticatorErrorDomain;
-        [Field ("kBITAuthenticatorErrorDomain", "__Internal")]
-        NSString AuthenticatorErrorDomain { get; }
-    
-        // extern NSString *const kBITHockeyErrorDomain;
-        [Field ("kBITHockeyErrorDomain", "__Internal")]
-        NSString HockeyErrorDomain { get; }
-    }
-
-    [BaseType (typeof (NSObject))]
-    public partial interface BITHockeyBaseManager 
-    {
-        [Export("serverURL")]
-        string ServerUrl { get;set; }
-
-        // @property (assign, nonatomic) UIBarStyle barStyle;
-        [Export ("barStyle", ArgumentSemantic.Assign)]
-        UIBarStyle BarStyle { get; set; }
-
-        // @property (nonatomic, strong) UIColor * navigationBarTintColor;
-        [Export ("navigationBarTintColor", ArgumentSemantic.Strong)]
-        UIColor NavigationBarTintColor { get; set; }
-
-        // @property (assign, nonatomic) UIModalPresentationStyle modalPresentationStyle;
-        [Export ("modalPresentationStyle", ArgumentSemantic.Assign)]
-        UIModalPresentationStyle ModalPresentationStyle { get; set; }
-    }
-
-    [BaseType (typeof(NSObject))]
-    [DisableDefaultCtor]
-    public partial interface BITHockeyManager
-    {
-        [Static]
-        [Export("sharedHockeyManager")]
-        BITHockeyManager SharedHockeyManager { get;set; }
-
-        [Wrap ("WeakDelegate")][NullAllowed]
-        BITHockeyManagerDelegate Delegate { get; set; }
-
-        [Export ("delegate", ArgumentSemantic.Assign)][NullAllowed]
-        NSObject WeakDelegate { get; set; }
-
-        [Export("configureWithIdentifier:")]
-        void Configure(string appIdentifier);
-
-        [Export("configureWithIdentifier:delegate:")]
-        void Configure(string appIdentifier, [NullAllowed] NSObject managerDelegate);
-
-        [Export("configureWithBetaIdentifier:liveIdentifier:delegate:")]
-        void ConfigureWithIdentifier(string betaIdentifier, string liveIdentifier, [NullAllowed] NSObject managerDelegate);
-
-        [Internal]
+	// typedef NSString * (^BITLogMessageProvider)();
```

Comment:
You probably accidentally reformatted this with tabs instead of spaces. Real changes are lost for looking at this.

Reply:
Nevermind
Tip: Add `?w=1` to the URL to see the diff with whitespace ignored.

Reply:
Sry about that. Xamarin Studio's fault ;)